src/modules/common/components/TransactionSocketWrapper/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect } from 'react';
+import { useQueryClient } from '@tanstack/react-query';
+import { toast } from 'react-toastify';
+import client from 'src/utils/api/client';
+import { TRANSACTIONS, AUTH } from 'src/const/queries';
+import { MODULE_COMMANDS_NAME_MAP } from '@transaction/configuration/moduleCommand';
+import { fromRawLsk } from '@token/fungible/utils/lsk';
+import i18n from 'src/utils/i18n/i18n';
+import { useCurrentAccount } from '@account/hooks';
+import { useCurrentApplication } from '@blockchainApplication/manage/hooks';
+import { useTransactions, useTransactionsConfig } from '@transaction/hooks/queries';
+
+const filterIncomingTransactions = (transactions, account) =>
```

Comment:
please move utility function to utils directory

Reply:
@soroushm Done